packages/ReactDagEditor/stories/components/FeaturesDemo.tsx

```diff
@@ -10,16 +10,16 @@ import {
   GraphModel,
```

Comment:
Please update the doc: remove the GraphStateStore, RegisterNode/Edges/Ports docs and add docs to describe useGraphReducer and state contracts.

Reply:
updated docs